core/src/com/mygdx/game/client/util/MovementUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.mygdx.game.client.util;
+
+import com.badlogic.ashley.core.ComponentMapper;
+import com.badlogic.ashley.core.Entity;
+import com.mygdx.game.client.component.PositionComponent;
+import com.mygdx.game.client.component.SlotComponent;
+
+public class MovementUtil {
+
+  private static final ComponentMapper<SlotComponent> slotComponentMapper = ComponentMapper.getFor(SlotComponent.class);
+  private static final ComponentMapper<PositionComponent> positionComponentMapper = ComponentMapper.getFor(PositionComponent.class);
+
+  public void moveUnit(Entity oldFieldEntity, Entity newFieldEntity) {
```

Comment:
A nie powinno takie coś być bardziej robione przez system i jakiś komponent "MovementTargetComp" czy coś?
Wtedy do entity unita wstawiany byłby ten component i system przechodziłby przez jednostki tak oznaczone i by je przesuwał. Teraz ta implementacja, która jest zamyka nas w w tym, że jest tylko jedno "entity" w slot componencie.

(W ogóle, to może skróćmy Component do Comp, bo to długie jest.)